Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/Handlers/SceneDebugConsoleReportHandler.cs

```diff
@@ -9,7 +9,7 @@ public class SceneDebugConsoleReportHandler : ReportHandlerBase
     {
         private readonly SceneDebugConsole sceneDebugConsole;
 
-        public SceneDebugConsoleReportHandler(ICategorySeverityMatrix matrix, bool debounceEnabled) : base(matrix, debounceEnabled)
+        public SceneDebugConsoleReportHandler(ICategorySeverityMatrix matrix, bool debounceEnabled) : base(ReportHandler.None, matrix, debounceEnabled)
```

Comment:
Shouldnt it be `REportHandler.DebugLog`? When this is initialized in `DiagnosticsContainer`, its key is `DebugLog`

Reply:
Good catch!